---
layout: default
title:  'Gymrek Lab - Resources'
---

<h4>Check out our lab <a href="https://github.com/gymreklab" target="_blank">github page</a> for more info on our software projects.</h4>

<h2>Tools for analyzing short tandem repeat variation</h2>
<table class="softwaretable">
  <tbody>
    <tr>
      <td width=50%>
	<h2>GangSTR</h2>
	GangSTR is a tool for genome-wide profiling tandem repeats from short reads. A key advantage of GangSTR over lobSTR and HipSTR is that it can handle repeats that are longer than the read length.
	<br>
	[<a href="https://www.biorxiv.org/content/early/2018/07/03/361162" target="_blank">paper</a>]
	[<a href="https://github.com/gymreklab/gangstr" target="_blank">github</a>]
      </td>
      <td width=50%>
	<h2>STRToolkit</h2>
	Our STRToolkit contains a variety of utilities for working with TR genotype files generated by HipSTR or GangSTR. These include:
	<ul>
	  <li>DumpSTR: a tool for filtering TR VCF files</li>
	  <li>PlinkSTR: a tool for performing association studies with TRs</li>
	</ul>
	<br>
	[<a href="https://github.com/gymreklab/STRTools" target="_blank">github</a>]
      </td>
    </tr>
    <tr>
      <td width=50%>
	<h2>WebSTR</h2>
	WebSTR (<a href="http://webstr.gymreklab.com">webstr.gymreklab.com</a>) is a web application for browsing a variety of STR datasets. It is based off of the popular ExAC browser.
	<br>
	[<a href="http://webstr.gymreklab.com" target="_blank">WebSTR</a>]
	[<a href="https://github.com/gymreklab/dbstr" target="_blank">github</a>]
      </td>
      <td width=50%>
	<h2>CookieMonSTR</h2>
	CookieMonSTR is a tool for calling de novo mutations from HipSTR or GangSTR VCF files. (<em>under construction</em>)
	<br>
	[<a href="https://github.com/gymreklab/strdenovotools" target="_blank">github</a>]
      </td>

    </tr>
    <tr>
      <td width=50%>
	<h2>PyBamView</h2>
	PyBamView is a python-based application that allows you to view BAM sequence alignments through your web browser. It was inspired by the samtools tview tool, and was motivated by the need to better visualize alignments with complex insertions/deletions across multiple samples, which is not handled well by current broswers such as IGV and the UCSC Genome Browser.
	<br>
	[<a href="http://www.ncbi.nlm.nih.gov/pubmed/25147359" target="_blank">paper</a>]
	[<a href="http://melissagymrek.com/pybamview/" target="_blank">website</a>]
	[<a href="https://github.com/mgymrek/pybamview" target="_blank">github</a>]
      </td>
      
    </tr>
    <tr>
      <td width=50%>
	<h2>lobSTR</h2>
	lobSTR is a tool for profiling Short Tandem Repeats (STRs) from high throughput sequencing data. It takes as input raw sequencing reads in either BAM or FASTQ format and outputs STR genotypes in a VCF file. lobSTR can be used to simultaneously genotype thousands of samples.
	<br>
	[<a href="http://www.ncbi.nlm.nih.gov/pubmed/22522390" target="_blank">paper</a>]
	[<a href="lobstr.teamerlich.org" target="_blank">website</a>]
	[<a href="https://github.com/mgymrek/lobstr-code" target="_blank">github</a>]
      </td>
    </tr>
</tbody>
</table>


<h2>Data</h2>

<h4>*** A reference haplotype panel for genome-wide imputation of short tandem repeats***</h4>
Information about our genome-wide SNP-STR haplotype panel is described <a href="https://gymreklab.github.io/2018/03/05/snpstr_imputation.html">here</a>.
<br><br><br>

<h4>*** A framework to interpret short tandem repeat variation in humans***</h4>
<ul>
  <li>Supplemental Dataset 1
  	<ul>
  		<li><a href="https://s3-us-west-2.amazonaws.com/strconstraint/Gymrek_etal_SupplementalData1.bed.gz">Gymrek_etal_SupplementalData1.bed.gz</a></li>
		<li><a href="https://s3-us-west-2.amazonaws.com/strconstraint/Gymrek_etal_SupplementalData1.bed.gz.tbi">Gymrek_etal_SupplementalData1.bed.gz.tbi</a></li>
		<li><a href="https://s3-us-west-2.amazonaws.com/strconstraint/readme.txt">readme.txt</a></li>
  	</ul>
  </li>
  <li>Code
  <ul>
  <li><a href="https://github.com/gymreklab/mutea-autosomal">mutea-autosomal github repository</a></li>
  </ul>
  </li>
</ul>
<br><br><br>

